agent/eni/watcher/watcher_linux.go

```diff
@@ -87,23 +50,12 @@ type UdevWatcher struct {
 	ctx                  context.Context
 	cancel               context.CancelFunc
 	updateIntervalTicker *time.Ticker
-	netlinkClient        netlinkwrapper.NetLink
-	udevMonitor          udevwrapper.Udev
-	events               chan *udev.UEvent
 	agentState           dockerstate.TaskEngineState
 	eniChangeEvent       chan<- statechange.Event
 	primaryMAC           string
-}
-
-// unmanagedENIError is used to indicate that the agent found an ENI, but the agent isn't
-// aware if this ENI is being managed by ECS
-type unmanagedENIError struct {
-	mac string
-}
-
-// Error returns the error string for the unmanagedENIError type
-func (err *unmanagedENIError) Error() string {
-	return fmt.Sprintf("udev watcher send ENI state change: eni not managed by ecs: %s", err.mac)
+	netlinkClient        netlinkwrapper.NetLink
```

Comment:
Why have these lines been updated?

Reply:
The common code has been moved to common platform independent files and Linux specific code is present in Linux files.